pandas_gbq/gbq.py

```diff
@@ -912,6 +927,9 @@ def read_gbq(
         credentials=credentials,
         private_key=private_key,
         use_bqstorage_api=use_bqstorage_api,
+        auth_redirect_uri=auth_redirect_uri,
```

Comment:
There are other uses of `GbqConnector` that need to be updated. Please add these auth argument to `to_gbq` as well.

Reply:
done